data/rupeshexe.json

```
@@ -28,6 +33,63 @@
       "name": "LinkedIn: Let's connect",
       "url": "https://linkedin.com/in/rupeshexe",
       "icon": "FaLinkedin"
+    },
+    {
+      "name": "YouTube: First Video Soon",
+      "url": "https://youtube.com/@rupeshexe",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "name": "Open Source and Random Tech Blogs",
+      "url": "https://rupeshexe.hashnode.dev/",
+      "icon": "SiHashnode",
+      "color": "#0755AA"
+    },
+    {
+      "name": "GitHub Sponsors: Support my Work",
+      "url": "https://github.com/sponsors/rupeshexe",
+      "icon": "FaDollarSign"
+    },
+    {
+      "name": "Mail me at",
+      "url": "mailto:contactrupeshdubey@gmail.com",
+      "icon": "SiGmail"
```

You missed a comma `,` again in the Mail link you just added. 

```suggestion
      "icon": "SiGmail",
```